cblaster/hmm_search.py

```diff
@@ -131,7 +134,13 @@ def write_profiles(profiles: Collection[str], output: str=None) -> str:
         output: name of output file
     """
     if not output:
-        output = datetime.now().strftime("cblaster_%Y%m%d%H%M%S.hmm")
+        counter = 0
+        output = Path(cagecat_prefix, datetime.now().strftime(f"cblaster_%Y%m%d%H%M%S-{counter}.hmm"))

```

Comment:
Can we generalise this and take out the hardcoded CAGECAT stuff here? (e.g. add an output folder to the arguments for `write_profiles`)